spec/close-tag-spec.coffee

```diff
@@ -26,11 +26,22 @@ describe 'closeTag', ->
       tags = tagFinder.tagsNotClosedInFragment(fragment)
       expect(tags).toEqual(['html', 'body', 'h1'])
 
+    # Is this desirable? It results in malformed XML for closing tag insertions.
```

Comment:
`it 'detects an incomplete tag'` is for preexisting behavior
`it 'is not confused by namespace prefixes'` is for the new behavior